src/idea.js

```diff
@@ -12,7 +12,12 @@ class Idea {
 
   }
   updateIdea() {
-
+    this.star = !this.star;
+    if (this.star) {
+      return "assets/star-active.svg";
+    }
+    else {
+      return "assets/star.svg";
+    }
```

Comment:
Had a new idea here and think it may be better practice. Instead of returning these strings to javascript to then reassign the HTML, maybe just reassign the HTML in this method.